src/containerapp/azext_containerapp/_params.py

```diff
@@ -24,8 +25,9 @@ def load_arguments(self, _):
         c.argument('artifact', help="Local path to the application artifact for building the container image. See the supported artifacts here: https://aka.ms/SourceToCloudSupportedArtifacts.", is_preview=True)
 
     # Springboard
-    with self.argument_context('containerapp create', arg_group='Service Binding') as c:
+    with self.argument_context('containerapp create', arg_group='Service Binding', is_preview=True) as c:
         c.argument('service_bindings', nargs='*', options_list=['--bind'], help="Space separated list of services(bindings) to be connected to this app. e.g. SVC_NAME1[:BIND_NAME1] SVC_NAME2[:BIND_NAME2]...")
+        c.argument('customized_keys', options_list=['--customized-keys'], action=AddCustomizedKeys, nargs='*', help='The customized keys used to change default configuration names. Key is the original name, value is the customized name.')
```

Comment:
There is no need to explicitly define the default option name

```suggestion
        c.argument('customized_keys', action=AddCustomizedKeys, nargs='*', help='The customized keys used to change default configuration names. Key is the original name, value is the customized name.')
```